test/utils.test.js

```diff
@@ -0,0 +1,59 @@
+const helper = require('../utils.js')
+
+const SECONDS_IN_DAY = 86400
+
+contract('Test Utils', async () =>  {
+
+    it("Test advanceBlock", async() => {
+        const currentBlock = await web3.eth.getBlock('latest')
+        const currentBlockNum = currentBlock.number
+
+        await helper.advanceBlock()
+        
+        const advancedBlock = await web3.eth.getBlock('latest')
+        const advanceBlockNum = advancedBlock.number
+        
+        assert.equal(currentBlockNum + 1, advanceBlockNum, "New block was not mined")
+    })
+
+    it("Test advanceTime", async() => {
+        const currentBlock = await web3.eth.getBlock('latest')
+
+        await helper.advanceTime(SECONDS_IN_DAY);
+        await helper.advanceBlock() // advanced time does not show unless block has been mined and shown in new block
```

Comment:
`advanced time does not show unless block has been mined and shown in new block`
Change to
`time doesn't update unless a block is mined`
Place the comment above this line.

Reply:
I agree with this